ketcher-autotests/playwright.config.ts

```diff
@@ -32,7 +46,7 @@ const MAX_NUMBER_OF_RETRIES = 2;
 const config: PlaywrightTestConfig = {
   testDir: './tests',
   /* Maximum time one test can run for. */
-  timeout: 60_000,
+  timeout: 120_000,
```

Comment:
I propose to left the timeout as 1 minute

Reply:
done